### ISSUE TITLE
what is the hyper-parameter in train process

### ISSUE THREAD

**Issue by ScarfBoy0903** [MuggleWang/CosFace_pytorch#7]
what is the batch size and the learning rate , decay rate you set
and have you try Adam optimizer?

**Comment by MuggleWang** [comment]
Default setting in `main.py`
Only used SGD
